runtime/doc/editing.txt

```diff
@@ -1206,9 +1206,16 @@ use has("browsefilter"): >
 ==============================================================================
 7. The current directory				*current-directory*
 
-You may use the |:cd| and |:lcd| commands to change to another directory, so
-you will not have to type that directory name in front of the file names.  It
-also makes a difference for executing external commands, e.g. ":!ls".
+You may use the |:cd|, |:tcd| and |:lcd| commands to change to another
+directory, so you will not have to type that directory name in front of the
+file names.  It also makes a difference for executing external commands, e.g.
+":!ls".
+
+There are three levels of working directories: global, tab and window.  The
+local working directory of the current window takes precedence over the local
+working directory of the tab, which in turn takes precedence over the global
+working directory.  If a local working directory, tab or window, does not
+exist, the next one in the hierarchy applies.
 
 Changing directory fails when the current buffer is modified, the '.' flag is
 present in 'cpoptions' and "!" is not used in the command.
```

Comment:
Yes, we do. Should we just silently ignore the exclamation mark in the documentation, pretending it doesn't exist, or write a note that it doesn't do anything? I'd go for the latter.